src/main/java/se/sundsvall/webmessagecollector/service/MessageService.java

```diff
@@ -35,8 +32,8 @@ public MessageService(final MessageRepository repository, final MessageAttachmen
 		this.attachmentRepository = attachmentRepository;
 	}
 
-	public List<MessageDTO> getMessages(final String familyId) {
-		return toMessageDTOs(repository.findAllByFamilyId(familyId));
+	public List<MessageDTO> getMessages(final String familyId, final String instance) {
+		return toMessageDTOs(repository.findAllByFamilyIdAndInstance(familyId, Instance.fromString(instance)));
 	}
```

Comment:
We should probably validate the instance value. With this current setup i'm guessing an invalid string will always return an empty list. Which might be OK?

Reply:
I don't know if it is OK. Just a heads up.